lint.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
-FILES_CHANGED=$(git diff --name-only HEAD~ HEAD | grep '.js')
-$(./node_modules/eslint/bin/eslint.js ${FILES_CHANGED} --ext .js --fix)
\ No newline at end of file
+# FILES_CHANGED=$(git diff --name-only HEAD~ HEAD | grep '.js')
+# $(./node_modules/eslint/bin/eslint.js ${FILES_CHANGED} --ext .js --fix)
```

Comment:
@hondahmd don't leave outcommented code in pushed code

Reply:
@adamgyulavari roger that, deletion done